### PR TITLE
feat: completed wishlist page for e-commerce

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
image

### CODE REVIEW

**Review by psychoactiv — APPROVED** [review]
Good to merge

File: pages/wishlist.html [hunk 1]
```diff
@@ -5,38 +5,125 @@
...
+                    <i class="fa-solid fa-user"></i>
+                </a>
             </li>
         </ul>
     </header>
-
+    <div class="body-container">
+        <div class="main-container">
+            <div class="product-card-container">
+                <img src="https://i.ibb.co/KL8Ly0q/3.jpg" class="product-card_container-images">
```
Comment (sameep-s) [inline pages/wishlist.html:48]: you can give alt tag to images, it's a good practice.

**Review by sameep-s — APPROVED** [review]
Code looks good just look at the comments.

File: pages/wishlist.html (hunk 1 above)
Comment (ajit-sharmaGH) [reply pages/wishlist.html:48]: Ohh, thank you I missed that almost in every img tag

**Pull request merged**